shared/src/main/scala/core/board/Board.scala

```diff
@@ -0,0 +1,12 @@
+package core.board
+
+import breeze.linalg.DenseMatrix
+import core.settings
+
+class Board {
+
+  val rows = settings.board.getInt("rows")
+  val cols = settings.board.getInt("cols")
```

Comment:
better in the constructor and companion object using those as default for zero arg constructor.